### DISCUSSION TITLE
Set Permmison

### DISCUSSION THREAD

**Discussion by DjPercy** [beeware/briefcase#694] [Show and tell]
Hello,
How can I set access rights in Briefcase? I need access to paths / files and the camera
when i run my APp i get the following error message
04-01 17:30:35.029  3167  3167 E Python  :  PermissionError: [Errno 13] Permission denied: '/data/.kivy'

Greeting , Denis

**Comment by freakboy3742** [comment]
There's no Briefcase-level setting for permissions at present - that's logged as feature request #547.

In the meantime, the solution is to use the methods provided by each platform, by modifying the files generated by the template after they've been created. 

If you want to persist those changes so they aren't lost between template runs, you can fork the template, and add a `template` setting in your `pyproject.toml` that points to your custom template.
